app/views/shared/_rate.html.erb

```diff
@@ -0,0 +1,9 @@
+<% rate.times do %>
+  <span class="icon-star text-warning"></span>
+<% end %>
+<% (Settings.rate_max-rate).times do %>
```

Comment:
tại sao phải thêm ()

Reply:
Tại vì trước đó em debug code bằng view ạ :(

Reply:
vì nó là toán tử (5 - rate) ạ

Reply:
Em đã fix rồi.